tldw_Server_API/app/services/admin_uptime_history_service.py

```
@@ -0,0 +1,137 @@
+                (service_name, status, latency_ms, error_message, checked_at)
+            VALUES (?, ?, ?, ?, ?)
+            """,
+            (service_name, status, latency_ms, error_message, now),
+        )
+
+    async def get_uptime_history(
+        self,
+        service_name: str,
+        *,
+        range_days: int = 30,
+        bucket_hours: int = 24,
```

Ruff (0.15.7)

[warning] Unused method argument: `bucket_hours`

(ARG002)